data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,450 @@
+			`	"Beware, for there are no rules, other than the fact you may only use this one ship. Tell me then, what say you, will you take to the skies, or are you too afraid?"`
+			choice
+				`	"I accept."`
+				`	"I have nothing to prove to you."`
+					decline
+			`	"Good! I will of course not be fighting you personally, as it is too risky to do so. I would also not want to deny my brethren the opportunity to prove themselves against a mighty human warrior!"`
+			`	You are not sure if they are being serious or not, but their friends seem to be in for it. Their leader approaches you. "So you think you deserve to fly this mighty ship of the Hai empire?"`
+			`	"Yeah, what makes you think that, human?" Another one adds.`
+			choice
+				`	(Say nothing.)`
+				`	"Are you trying to scare me out of the fight?"`
+					goto scare
+			`	"I guess we'll find out." Says a third.`
+			label scare
+			`	The leader gets closer and addresses you in a more serious tone, "You should be aware, <last>, that this is not a friendly competition between allies. As far as I am concerned, you are an enemy. You should see it more as a fight for you to truly earn this ship, and more, but make no mistakes: it has only two rules: nobody else than us may interfere, and being disabled results in losing the fight. And do not even think about bringing more than this Ladybug as a flagship or we will not bother showing up!"`
+			# this is a "warning" for the incoming extra ship that happens because you killed/captured 10 Shield Beetles
+			# (but in reality with disabling a lot of ships it goes faster than that)
+			`	"Sometimes the rules can be bent a little. But you certainly did not earn that privilege."`
+				to display
+					"reputation: Hai (Unfettered)" < -2000
+			`	The engineer comes back, looking a bit impatient, "Are you going to talk all day, or are you going to fight?" The Unfettered pilots immediately head to their ships to get the initiative. The engineer utters a word to you once they are far enough away, "I want this done by the end of the day, do not try my patience. You are lucky I offered this deal in the first place, make sure I do not regret it. Now, quick, get to your ship!" You see the contestants have already begun the launching procedures, and launch as soon as you can.`
+			`	(Remember to park your other ships and make sure your flagship is the Ancient Ladybug.)`
+				accept
+	on decline
+		"reputation: Hai (Unfettered)" >?= -1000
+	on disabled
+		fail "Hai Legacy 3"
+		dialog `The enemy ships hail you, "You have failed the challenge, we will take the technology we want when you land, and have already arranged to sell you replacements for it, for this fine of <fine>."`
+		fine 1500000
```

Comment:
Maybe wait until your ship taking PR (#6484) is merged and then take the Ladybug?

Reply:
yeah that's totally what I was planning to do, but it works like this in the meantime, and the dialog sort of makes sense with it
I'll update this mission whenever that one is merged, we'll see which one goes in first

Reply:
done!